perl-libs/packaging/centreon-perl-libs-common.yaml

```diff
@@ -54,6 +54,12 @@ overrides:
       - libdigest-sha-perl
       - libgd-perl
       - libcrypt-urandom-perl
+    replaces:
+      - centreon-perl-libs (<< 25.10.0)
```

Comment:
~~Wondering out loud: `(<< 25.10.0)` is a deb-specific syntax, but the Replaces instruction is not in a deb-specific part of the file. Not sure what it will do for the rpm that will be built.~~

It is in a deb-specific part